test/viewport.test.js

```diff
@@ -54,4 +84,19 @@ describe('o-viewport', function() {
 	it('should get the orientation of the viewport', function() {
 		expect(oViewport.getOrientation() === 'portrait' || oViewport.getOrientation() === 'landscape').to.be(true);
 	});
+
+	it('should stop listening to scroll event', function(done) {
+		oViewport.stopListeningTo('scroll');
+		document.body.addEventListener('oViewport.scroll', function(ev) {
+			expect(false).to.be.true;
```

Comment:
Forgot I was supposed to look up how to do that properly, I've updated it
now.


On 24 March 2015 at 15:58, Kaelig <notifications@github.com> wrote:

> In test/viewport.test.js
> <https://github.com/Financial-Times/o-viewport/pull/17#discussion_r27042323>
> :
>
> > @@ -54,4 +84,19 @@ describe('o-viewport', function() {
> >  	it('should get the orientation of the viewport', function() {
> >  		expect(oViewport.getOrientation() === 'portrait' || oViewport.getOrientation() === 'landscape').to.be(true);
> >  	});
> > +
> > +	it('should stop listening to scroll event', function(done) {
> > +		oViewport.stopListeningTo('scroll');
> > +		document.body.addEventListener('oViewport.scroll', function(ev) {
> > +			expect(false).to.be.true;
>
> [image: quote-william-s
> -burroughs-nothing-is-true-everything-is-permitted-124339_1]
> <https://cloud.githubusercontent.com/assets/85783/6806103/aca69098-d23e-11e4-8c7e-81a4c5560005.png>
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Financial-Times/o-viewport/pull/17/files#r27042323>.
>

-- 

------------------------------
This email was sent by a company owned by Pearson plc, registered office at 
80 Strand, London WC2R 0RL.  Registered in England and Wales with company 
number 53723.